extension-manifest-v3/src/background/utils/startup-guard.js

```diff
@@ -0,0 +1,56 @@
+export default (starter, stopper) => {
```

Comment:
Please use the `function` statement in modules where possible (if you don't set `const something = () => {}`. Also, the copyright missing.